### PR TITLE
Feature/waivers

### PR DESCRIPTION
**Pull request by davey-gov**

### CODE REVIEW

File: _includes/waiver-teaser.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    margin-y-105
+    border border-base-lighter
+    radius-sm
+    text-base-darker
+  "
+  style="justify-content: space-between; max-width: none"
+>
+  <div class="tablet:grid-col-3">
+    <h3 class="margin-0">Procurrement Title</h3>
+    <p class="margin-0 text-primary-vivid">{{ include.title }}</p>
```
Comment (scottqueen-bixal) [inline _includes/waiver-teaser.html:15]: In Prototype this links to call the modal as well. It seems like a strange interaction to have two refs in the same card toggle the modal? Can we confirm that it is needed?

File: _includes/modal.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
+<div
+  class="usa-modal maxw-desktop"
```
Comment (scottqueen-bixal) [inline _includes/modal.html:2]: I didn't see the "x" here for closing the modal, looks like clicking outside the element does close it.

File: _includes/modal.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    <div class="usa-modal__main width-full">
+      <h2
+        class="usa-modal__heading text-primary-vivid font-sans-xl"
+        id="{{include.id}}-heading"
+      >
+        {{include.title}}
+      </h2>
+      <div class="text-base-darker font-sans-sm margin-bottom-2">
+        <h3 class="margin-0">Procurement Synopsis</h3>
+        <p class="margin-0">{{include.synopsis}}</p>
```
Comment (scottqueen-bixal) [inline _includes/modal.html:17]: Do we want a fall back here if data not there? Like, "N/A" or do we just not display the label?

File: search/index.html [hunk 4]
```diff
@@ -4,70 +4,117 @@
...
+      </div>
+    </div>
   </div>
 </div>
 
-<ul id="results-container" style="padding: 0"></ul>
-
-<!-- or without installing anything -->
+<ul id="results-container" style="padding: 0">
+  {% include waiver-teaser.html id="cripps-pink" title='Cripps Pink / Pink Lady
```
Comment (scottqueen-bixal) [inline search/index.html:47]: Have we considered using collections here? https://github.com/GSA/made-in-america/blob/main/_data/collections.yaml if we were able to manage the content outside of the template code, there would be less friction for content updating the yml and submitting PR? I know we only have the five at the moment but we will be likely updating manually until the integration occurs, trying to see if that task can be off loaded...

File: search/index.html [hunk 5]
```diff
@@ -4,70 +4,117 @@
...
-    limit: 10,
-    fuzzy: false,
-    exclude: ["Welcome"],
-  });
-</script>
+  <script src="https://unpkg.com/simple-jekyll-search@latest/dest/simple-jekyll-search.min.js"></script>
+  <script>
+    window.simpleJekyllSearch = new SimpleJekyllSearch({
+      searchInput: document.getElementById("search-input"),
+      resultsContainer: document.getElementById("results-container"),
```
Comment (scottqueen-bixal) [inline search/index.html:110]: Can we hide search and sort so it doesn't cause any misunderstanding from what will be in the MVP 1 release?

File: _includes/modal.html (hunk 3 above)
Comment (davey-gov) [reply _includes/modal.html:17]: We don't but definitely something we should think about in the next implementation

File: _includes/waiver-teaser.html (hunk 1 above)
Comment (davey-gov) [reply _includes/waiver-teaser.html:15]: Lets discuss during sync

**Pull request merged**